rolling/distribution.yaml

```diff
@@ -1404,7 +1404,7 @@ repositories:
       tags:
         release: release/rolling/{package}/{version}
       url: https://github.com/ros2-gbp/hpp_fcl-release.git
-      version: 2.1.2-2
+      version: 2.1.2-3
```

Comment:
I have the same question here https://github.com/ros/rosdistro/pull/34051#pullrequestreview-1062687309